packages/@webex/plugin-meetings/test/unit/spec/meeting/index.js

```diff
@@ -3802,281 +3906,292 @@ describe('plugin-meetings', () => {
             sendShare: false,
           };
           meeting.isMultistream = true;
-          meeting.mediaProperties.webrtcMediaConnection = {
-            publishTrack: sinon.stub().resolves({}),
-            unpublishTrack: sinon.stub().resolves({}),
-          };
-          meeting.audio = { handleLocalTrackChange: sinon.stub()};
-          meeting.video = { handleLocalTrackChange: sinon.stub()};
-
-          const createFakeLocalTrack = (originalTrack) => ({
-            on: sinon.stub(),
-            off: sinon.stub(),
-            stop: sinon.stub(),
-            originalTrack,
+          meeting.mediaProperties.webrtcMediaConnection = {};
+          meeting.audio = { handleLocalStreamChange: sinon.stub()};
+          meeting.video = { handleLocalStreamChange: sinon.stub()};
+          meeting.sendSlotManager = new SendSlotManager({
+            logger: {
+              info: () => undefined
+            }
           });
+          fakeMultistreamRoapMediaConnection = {
+            createSendSlot: sinon.stub().returns({
+              publishStream: sinon.stub(),
+              unpublishStream: sinon.stub(),
+            })
+          }
+          meeting.sendSlotManager.createSlot(fakeMultistreamRoapMediaConnection, MediaType.VideoSlides);
+          meeting.sendSlotManager.publishStream = sinon.stub();
+          meeting.sendSlotManager.unpublishStream = sinon.stub();
+          meeting.sendSlotManager.setActive = sinon.stub();
 
           // setup mock constructors for webrtc-core local track classes in such a way
           // that they return the original track correctly (this is needed for unpublish() API tests)
-          LocalDisplayTrackConstructorStub = sinon
-            .stub(InternalMediaCoreModule, 'LocalDisplayTrack')
+          LocalDisplayStreamConstructorStub = sinon
+            .stub(InternalMediaCoreModule, 'LocalDisplayStream')
             .callsFake((stream) => {
-              fakeLocalDisplayTrack = createFakeLocalTrack(stream.getTracks()[0]);
-              return fakeLocalDisplayTrack;
+              return stream
             });
-          LocalMicrophoneTrackConstructorStub = sinon
-            .stub(InternalMediaCoreModule, 'LocalMicrophoneTrack')
+          LocalSystemAudioStreamConstructorStub = sinon
```

Comment:
I'm slightly confused, this is something I don't understand: why this PR shows that you had to **add** things for system audio stream? is the stream-classes branch based on SDK beta version that already has share audio? If yes, then in this case these tests should already take share audio into account and you would only need to **modify** them not add new ones.

Reply:
In the beta, I do not see a stub for the Share Audio track: https://github.com/webex/webex-js-sdk/blob/35b946c2ef436248073043eb73c79e3af6e25d81/packages/%40webex/plugin-meetings/test/unit/spec/meeting/index.js

Am I missing something?

Reply:
looking at this a bit more, I think these constructor stubs are actually not needed at all, because these tests are using fake track (or now stream) objects like audioTrack, videoTrack, audioShareTrack, videoShareTrack created at the beginning of beforeEach(), so the constructors are never called. If I remove these constructor stubs in beta branch, these tests still pass, so you can just remove them

Reply:
Done. Removed and tests are passing. Thanks.